timeplus_connect/cc_sqlalchemy/dialect.py

```diff
@@ -10,7 +11,6 @@
 from timeplus_connect.cc_sqlalchemy.sql.preparer import TpIdentifierPreparer
 from timeplus_connect.driver.binding import quote_identifier, format_str
 
-
```

Comment:
let's add this empty line back. Minor thing but will make our future rebase/cherrypick a bit easier